basics/answers.md

```diff
@@ -92,7 +92,18 @@ isn't exactly like mine but it works just fine it's ok, and you can
         print("Access denied.")
     ```
 
-    Again, this is not a good way to ask a real password from the user.
+    Again, this is not a good way to ask a real password from the user. 
+
+7. Here, we create  a new variable `pallindrome_check`.We use this variable to store the  reversed  string(of`pallindrome_input`). <br>Then we check if the string that was given(`pallindrome_input`)
+   and the reversed sting(`pallindrome_check`) is the same using a simple `if loop`
+   ```python
+   pallindrome_input=input("Type the number to check:") #to get the input from user
```

Comment:
Don't write comments like this. The tutorial explains that `input(...)` gets input from the user. There is no need to explain it every time we use the function. If you want to use something that isn't explained in the tutorial, then add something to the tutorial explaining it. Most of your other comments are similar.

However, you should add comments to mention why you need to add `+2` when doing `rows+2`. That's not obvious without a comment.